.Dockerfiles/ubuntu/rolling/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update \
 	python2.7 \
 	sudo \
 	systemd \
- && pip install --upgrade pip \
+ && pip install --upgrade pip==10.0.1 \
```

Comment:
Why do we have such constraint? 

It is better not to pin versions like this. Since when any updates (including security ones) will be released we won't have them here.

Reply:
Something I read made me think this might fix the PBR issue: #157